saleor/payment/gateways/utils.py

```diff
@@ -8,12 +9,15 @@
 
 
 def get_supported_currencies(config: "GatewayConfig", gateway_name: str) -> List[str]:
-    currencies = config.supported_currencies
-    if not currencies:
+    supp_currencies = config.supported_currencies
+    if not supp_currencies:
         currencies = [settings.DEFAULT_CURRENCY]
         warnings.warn(
             f"Default currency used for {gateway_name}. "
             "DEFAULT_CURRENCY setting is deprecated, "
             "please configure supported currencies for this gateway."
         )
+    else:
+        currencies = re.split(r",\s*", supp_currencies)
```

Comment:
I assume this regexp is used to trim spaces? It won't work for strings like `" USD, PLN "`

A bit easier and it always works:
```
currencies = [c.strip() for c in supp_currencies.split(",")]
```